swift/ql/src/queries/Security/CWE-321/HardcodedEncryptionKey.ql

```diff
@@ -3,7 +3,7 @@
  * @description Using hardcoded keys for encryption is not secure, because potential attackers can easily guess them.
  * @kind path-problem
  * @problem.severity error
- * @security-severity 8.1
+ * @security-severity 7.7
```

Comment:
Did you intend to commit this change?

Reply:
Yes - I ran the (latest) `cwe-scores` script on the entire swift `src` directory, and it make this change in addition to scoring the new query.  So presumably the old value here was either incorrect, or is now out-of-date due to some change in the scoring data?

Reply:
The severity was most likely copied from similar queries (like the [C# one](https://github.com/github/codeql/blob/main/csharp/ql/src/Security%20Features/CWE-321/HardcodedEncryptionKey.ql)). I can imagine the severity was manually downgraded on some of the other languages because we believe 8.1 was too high.

In any case, I think we should be consistent wrt. the severity across languages, so it might be worth moving this to another PR (and then updating all of the similar queries), and we can then discuss it more widely.

Reply:
> I can imagine the severity was manually downgraded

I thought that was a no-no.  The whole point of `security-severity` is that it's objective?

> it might be worth moving this to another PR (and then updating all of the similar queries), and we can then discuss it more widely.

Will do.

Reply:
I've reverted that bit, and started a conversation on Slack about the right thing to do here.